ruff_dev/src/generate_rules_table.rs

```diff
@@ -25,14 +26,18 @@ fn generate_table(table_out: &mut String, prefix: &RuleCodePrefix) {
     table_out.push('\n');
     table_out.push_str("| ---- | ---- | ------- | --- |");
     table_out.push('\n');
-    for rule_code in prefix.codes() {
-        let kind = rule_code.kind();
-        let fix_token = if kind.fixable() { "🛠" } else { "" };
+    for rule in prefix.codes() {
+        let fix_token = match rule.autofixable() {
+            Autofixability::Never => "",
+            Autofixability::Sometimes => "🛠 (sometimes)",
+            Autofixability::Always => "🛠",
```

Comment:
I'd prefer to leave this for now. We should make it clearer eventually, but I worry that as-is it'll hurt the readability and simplicity of the table.

I'm happy to play around with some alternate formats in a future PR.

![Screen Shot 2023-01-19 at 10 26 25 AM](https://user-images.githubusercontent.com/1309177/213482810-feb07291-7dbd-49e1-a268-8e6c4c218c38.png)

Reply:
(Unrelatedly: the use of the actual format strings is not only simpler for maintenance but so much clearer for users. Nice job.)

Reply:
Fair enough I have removed the display of `(sometimes)`.